interface/src/app/signature/high_range_exp.tpl.html

```diff
@@ -2,12 +2,10 @@
   <h4>
     <b>Experiments with the Highest Range: </b>
   </h4>
-  <button class="btn btn-primary" ng-click="setMode()"
-          ng-hide="topMode">Show Top {{topNum}}</button>
-  <button class="btn btn-primary" ng-click="setMode()"
-          ng-show="topMode">Show All</button>
-
-  <br>
+  <p>
+    Explore gene expression experiments with the highest range of this
+    signature's activity.
+  </p>
   (Click the activity distribution widget to see sample annotations in this experiement.)
```

Comment:
Shouldn't this parenthetical text be wrapped in a `<p>` tag?

Also I noticed Casey's comment about the tables not quite looking consistent. This table doesn't need a header column since it only has one column, but it might help the look a bit to have a line break here so the text isn't almost on top of the table.